po/de.po

```diff
@@ -131,7 +131,7 @@ msgstr "Arbeitsspeicher:"
 #: src/Views/DiskView.vala:52
 #, fuzzy
```

Comment:
```suggestion
```

You need to remove this line, otherwise your translation won't be shown in the UI. This is a "needs-editing" flag to tell translators that this translation is just filled automatically with the similar one. That's why `the previous translation had a totally different meaning`.